test/unit/storage/entities/account.js

```diff
@@ -63,16 +59,14 @@ describe('Account', () => {
 		it('should throw error for in-valid filters');
 		it('should accept only valid options');
 		it('should throw error for in-valid options');
-		it(
-			'should call adapter.executeFile with proper param for FIELD_SET_SIMPLE'
-		);
-		it('should call adapter.executeFile with proper param for FIELD_SET_FULL');
+		it('should call adapter.executeFile with proper param for extended=false');
+		it('should call adapter.executeFile with proper param for extended=true');
 		it('should accept "tx" as last parameter and pass to adapter.executeFile');
 		it(
-			'should resolve with array of objects matching specification of type definition for FIELD_SET_SIMPLE'
+			'should resolve with one object matching specification of type definition of simple object'
```

Comment:
This is more of an integration test as in unit we're not using the DB then we'll be testing a mock is correct.

Reply:
Same as above.

Reply:
Same answer as above.